Makefile

```diff
@@ -23,74 +23,58 @@ stack-down: ## stop dsp-stack and remove the cloned dsp-api repository
 
 .PHONY: dist
 dist: ## generate distribution package
-	python3 setup.py sdist bdist_wheel
+	@rm -rf dist/ build/
+	poetry build
 
 .PHONY: upload
 upload: ## upload distribution package to PyPI
-	python3 -m twine upload dist/*
+	poetry publish
 
 .PHONY: docs-build
 docs-build: ## build docs into the local 'site' folder
-	mkdocs build
+	poetry run mkdocs build
 
 .PHONY: docs-serve
 docs-serve: ## serve docs for local viewing
-	mkdocs serve --dev-addr=0.0.0.0:7979
-
-.PHONY: install-requirements
-install-requirements: ## install Python dependencies from the diverse requirements.txt files
-	python3 -m pip install --upgrade pip
-	pip3 install -r requirements.txt
-	pip3 install -r docs/requirements.txt
-	pip3 install -r dev-requirements.txt
+	poetry run mkdocs serve --dev-addr=0.0.0.0:7979
 
 .PHONY: install
-install: ## install from source (runs setup.py)
-	python3 -m pip install --upgrade pip
-	pip3 install -e .
+install: ## install Poetry, which in turn installs the dependencies and makes an editable installation of dsp-tools
+	curl -sSL https://install.python-poetry.org | python3 -
+	poetry install
 
 .PHONY: test
 test: dsp-stack ## run all tests located in the "test" folder (intended for local usage)
-	-pytest test/	# ignore errors, continue anyway with stack-down
+	-poetry run pytest test/	# ignore errors, continue anyway with stack-down
```

Comment:
Are you sure that the `-` is correct like this (there are more such occurrences further down)?

Reply:
I checked it, and yes, it's correct. I also added the link to the manual where it is described